webapp/src/components/data/DataDescription.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import type { Task } from '@epfml/discojs'
+
+import DropdownCard from '@/components/containers/DropdownCard.vue'
+
+interface Props {
+  task: Task
+}
+const props = defineProps<Props>()
```

Comment:
consider not depending on `Task` but rather on `DisplayInformation` as too specify which part of this global is needed. also, I link inlining, and no need to bind it to `props`, you can direclty use it (`props.taks.displayinformation.[...] -> info.[...]`)

```suggestion
defineProps<{
  info: DisplayInformation
}>()
```

Reply:
Unfortunately there is some logic that depends on `trainingInformation` (to get the task data type)